modules/dlscript/godot.cpp

```diff
@@ -175,6 +175,16 @@ void GDAPI godot_script_register_signal(const char *p_name, const godot_signal *
 	library->_register_script_signal(p_name, p_signal);
 }
 
+void GDAPI *godot_dlinstance_get_userdata(godot_object *p_instance) {
+	Object *instance = (Object *)p_instance;
+	if (!instance)
+		return NULL;
+	DLInstance *script = (DLInstance *)instance->get_script_instance();
```

Comment:
And what if it isn't a DLInstance? The way you currently did it would probably crash or return vague numbers..